### ISSUE TITLE
After changing default cluster icon to custom icon, the cluster size is not drawn over the icon

### ISSUE THREAD

**Issue by Filip94** [mierzynskim/Xamarin.Forms.GoogleMaps.Clustering#36]
### VERSIONS

* Xamarin.Forms.GoogleMaps.Clustering - x.x.x
* Xamarin.Forms - x.x.x
* Android/iOS - x.x.x

### PLATFORMS

- [ ] Android
- [x] iOS

### ACTUAL BEHAVIOR

When is set the custom icon with methods SetRenderUsingImage or SetRenderUsingCustomAction the icon is displayed but the cluster text size is not drawn over the image, and as i saw in the code it says that it should automatically be drawn over the icon.
I even tried to create view, and than set that view, but instead of the view the default pin is rendered as a cluster. 
### SCREENSHOTS/STACKTRACE

![something_image](your_image.png) 

```
// paste stacktrace
```

### HOW TO REPRODUCE AND/OR EXAMPLE PROJECT

1. 
2. 
3.

**Comment by mierzynskim** [comment]
@Filip94 Thanks for pointing it out. Is it Android-only issue? It should work fine on iOS. It looks like it's not possible to draw over a marker on Android but I'll try to do more research on this. I'm also looking forward for any suggestions!

**Comment by Filip94** [comment]
yes, it's Android issue only, but i managed to fix it. The problem on iOS is that the font size isfixed and you cannot change it, and you cannot set to see the correct number of pins in the cluster. You can see only the numbers from the bucket, for example 10+, 50+ etc..
